Use `docker-compose.override.yml` for dev environment

siefkenj opened the pull request:
Docker will automatically pull in `docker-compose.yml`
	and `docker-compose.override.yml` when running `docker-compose
	up`. The default for `docker-compose up` should be to start
	the development environment.

	Also fixes a bug due to the an old `I18n` configuration.

freeatnet commented:
I'm having trouble filtering people by "course" in assigned / unassigned views. Not sure if this is a regression specific to this branch, though.

Otherwise, 💕 , this PR is a thing of beauty.

gabriellesc commented:
@freeatnet Do you mean that you have trouble using the UI to do it (clicking, etc.), that there is a latency issue with the filtering being applied, or that the filtering doesn't work properly?

freeatnet commented:
@gabriellesc I navigate to (un-) assigned view, see some people listed. I click on the course filter and select a course from the dropdown. The list changes, however, not all people in the list have the selected course in their preferences or assignments.

gabriellesc commented:
@freeatnet that's strange - I don't get that issue with assigned (although I haven't had a chance to test it with unassigned)
would you be able to give me a specific instance of filtering and the unexpected result?

gabriellesc commented:
Update (elsewhere) from @freeatnet:
> @gabrielle Checked it out. Everything works on current #133 and after rebuilding stuff. :+1: Awesomesauce!

siefkenj committed:
Use `docker-compose.override.yml` for dev environment

	Docker will automatically pull in `docker-compose.yml`
	and `docker-compose.override.yml` when running `docker-compose
	up`. The default for `docker-compose up` should be to start
	the development environment.

	Also fixes a bug due to the an old `I18n` configuration.

George334456 reviewed (approved):
Looks good to me.
